MaterialLibrary/src/main/res/values/attrs_elevation_delegate.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <declare-styleable name="ElevationDelegate">
+        <attr name="elevation" />
+        <attr name="cornerRadius" format="dimension" />
+        <attr name="showShadows">
```

Comment:
Pretty subjective, but maybe `shownShadows` or `shadowVisibility`? I'm not fond of the verb in an attribute name. :smile: